test/filesystems/blk_filesystem_test.rb

```diff
@@ -88,4 +88,28 @@
       end
     end
   end
+
+  describe "#in_network?" do
```

Comment:
If I create a file  `test/filesystems/base`, it would look like I'm testing the `Base` class, when in fact, these tests that you wrote would test subclasses of the `Base` class (`BlkFilesystem` and `NFS`). In the future, if I wanna see how I would expect the behavior of `BlkFilesystem`, I would have to go to the tests of the `Base` class.
I don't know if that's how we do it in YaST, but it seems strange to me...